<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>9</y>
     <width>291</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="RegForm" name="RegWidget">
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
   </widget>
   <widget class="LogForm" name="LogWiget">
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogForm</class>
   <extends>QWidget</extends>
   <header>logform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegForm</class>
   <extends>QWidget</extends>
   <header>regform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
